README.md

```diff
@@ -266,6 +266,14 @@ will run the single test found under VSO_0000000_any_calling_conventions.
 control over what category of tests you would like to run. The following will run all the libcxx map tests.
   + `python tests\utils\stl-lit\stl-lit.py ..\..\llvm-project\libcxx\test\std\containers\associative\map`
 
+* You can also use the `--filter` option to include tests whose names match a regular expression. The following
+  command will run tests with "atomic_wait" in their names in both the std and libcxx test suites.
+
+  + `python tests\utils\stl-lit\stl-lit.py ..\..\llvm-project\libcxx\test ..\..\tests\std --filter=atomic_wait`
+
+  There's also a `--filter-out` option to exclude tests matching a regular expression;
+  `--filter=iota --filter-out=view` would run tests with names matching "iota" but not "view".
+

```

Comment:
@CaseyCarter, I see why you introduced these empty lines (they allow the `--filter-out` sentence to follow the python command, instead of wrapping into that sub-bullet), but Markdown has special behavior when empty lines appear in lists. The entire list will become un-compact, which I believe was an unintentional change here. Compare:

```markdown
* Cute fluffy kittens
* On consecutive lines
* Displayed very compactly
```

is displayed as:

* Cute fluffy kittens
* On consecutive lines
* Displayed very compactly

Whereas:

```markdown
* Again starting with
* Consecutive lines here
* But now for something different

* Empty line above here

* Another empty line above here
```

is displayed as:

* Again starting with
* Consecutive lines here
* But now for something different

* Empty line above here

* Another empty line above here